test.sh

```diff
@@ -45,7 +45,7 @@ for sample in $(ls $INPUT); do
   rm -f measurements.txt
   ln -s $sample measurements.txt
 
-  diff <("./calculate_average_$FORK.sh" | ./tocsv.sh) <(./tocsv.sh < ${sample%.txt}.out)
+./compare_output.sh <("./calculate_average_$FORK.sh") ${sample%.txt}.out
```

Comment:
Can we make this parameterized instead, i.e. add a `--lenient` option to the script which controls which mode to use?